External/FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -6865,6 +6865,11 @@ void OpDispatchBuilder::X87BinaryOp(OpcodeArgs) {
   // Overwrite the op
   result.first->Header.Op = IROp;
 
+  if (IROp == IR::OP_F80FPREM) {
+    //TODO: Set C0 to Q2, C3 to Q1, C1 to Q0
```

Comment:
Can we expand on this and say something like: "FREP is not correctly implemented at all, but this hack is better than nothing, preventing infinite loops"

Reply:
Well, it does give correct results - as long as one doesn't use C0, C1 or C3 and doesn't depend on the iterative nature